src/site/pages/digital/src/utils/BrowserCompatibility.ts

```diff
@@ -0,0 +1,85 @@
+import {Browser} from "core/utils/Browser";
+import { type } from "os";
+
+type compatible = {
+    react_v: boolean;
+    canvas_v:boolean;
+}
+
+var COMPATIBILITY = {
+    CHROME: {
+        react_v: "49",
+        canvas_v: "94"
+    },
+    FIREFOX: {
+        react_v: "49",
+        canvas_v: "93"  
+    },
+    SAFARI: { 
+        react_v: "49",
+        canvas_v: "93"  
+    },
+    IE: {
+        react_v: "49",
+        canvas_v: "93"
+    }
+}
+
+export function in_range(value: String, min:string): boolean {
+    return value >= min;
+};
+
+export function compatibility_check() : compatible {
+    var ans:compatible = {react_v: false, canvas_v: false};
+    if(Browser.name == "Chrome") {
```

Comment:
You made a `Record` up above (`COMPATIBILITY`), which means if you have all the keys be like `"Chrome"` instead of `CHROME`, you can just do `COMPATIBILITY[Browser.name]` and then you won't need these 4 separate if-statements that are doing virtually the same thing